examples/3.routes/8.tolls-and-ferries/queries.js

```diff
@@ -0,0 +1,97 @@
+import qql from 'graphql-tag';
+/*
+ * In this example a route from Aalborg, Denmakr to Oslo, Norway is requested.
+ * Origin and destination are required fields. An EV also needs to be selected.
+ * Only the EV ID here is mandatory, all other fields are optional and when not specified will use the default values.
+ * The changing conditions are:
+ *   - 80% SoC at Aalborg, Denmark
+ *   - EV can charge at CHAdeMO changers
+ *   - should use climate (temperature and weather conditions)
+ *   - min power of chargers is 43 kWh. This is the default setting
+ *   - one passenger in the car (drive alone)
+ */
```

Comment:
occupant count is 2 in the  newRoute mutation

Reply:
We also switched from passengers to occupants so lets use that terminology here as well.

Reply:
good spot!